installation-bundle/src/Database/Installer.php

```diff
@@ -257,7 +257,7 @@ private function checkEngineAndCollation(array &$sql, array &$order, Schema $fro
                     $command = 'ALTER TABLE '.$tableName.' ENGINE = '.$engine.' ROW_FORMAT = DYNAMIC';
 
                     if (false !== stripos($tableOptions->Create_options, 'key_block_size=')) {
-                        $command .= ', KEY_BLOCK_SIZE=0';
+                        $command .= ', KEY_BLOCK_SIZE = 0';
```

Comment:
I’d remove the comma too 🤓
```suggestion
                        $command .= ' KEY_BLOCK_SIZE = 0';
```